styles.css

```diff
@@ -26,40 +26,30 @@ body {
 
 .a {
     grid-area: a;
-
-
 }
 
 .b {
     grid-area: b;
-
-
 }
 
 .c {
     grid-area: c;
-
 }
 
 .d {
     grid-area: d;
     min-height: 100px;
```

Comment:
Deze min-height bedoelde ik in de eerste opmerking. Hierdoor moet je deze opnieuw definiëren voor mobiel.